build_tools/bazel_to_cmake/bazel_to_cmake_targets.py

```diff
@@ -65,6 +65,63 @@
     "@llvm-project//mlir:TensorDialect": ["MLIRTensor"],
     "@llvm-project//mlir:NVVMDialect": ["MLIRNVVMIR"],
     "@llvm-project//mlir:ROCDLDialect": ["MLIRROCDLIR"],
+    # MHLO.
+    # TODO: Rework this upstream so that Bazel and CMake rules match up
+    # better.
+    # All of these have to depend on tensorflow::external_mhlo_includes to
+    # ensure that include directories are inherited.
+    "@mlir-hlo//:chlo_legalize_to_hlo": [
+        "tensorflow::external_mhlo_includes",
+        "ChloPasses",
+    ],
+    "@mlir-hlo//:hlo": [
+        "tensorflow::external_mhlo_includes",
+        "ChloDialect",
+        "MhloDialect",
+        "MLIRMhloUtils",
+    ],
+    "@mlir-hlo//:legalize_control_flow": [
+        "tensorflow::external_mhlo_includes",
+        "MhloToStandard",
+    ],
+    "@mlir-hlo//:legalize_einsum_to_dot_general": [
+        "tensorflow::external_mhlo_includes",
+        "MhloPasses",
+    ],
+    "@mlir-hlo//:legalize_gather_to_torch_index_select": [
+        "tensorflow::external_mhlo_includes",
+        "MhloPasses",
+    ],
+    "@mlir-hlo//:legalize_to_linalg": [
+        "tensorflow::external_mhlo_includes",
+        "MhloLhloToLinalg",
+    ],
+    "@mlir-hlo//:legalize_to_standard": [
+        "tensorflow::external_mhlo_includes",
+        "MhloToStandard",
+    ],
+    "@mlir-hlo//:map_lmhlo_to_scalar_op": [
+        "tensorflow::external_mhlo_includes",
+        "LmhloDialect",  # Unfortunate.
+        "MhloDialect",
+    ],
+    "@mlir-hlo//:materialize_broadcasts": [
+        "tensorflow::external_mhlo_includes",
+        "MhloPasses",
+    ],
+    "@mlir-hlo//:mhlo_control_flow_to_scf": [
+        "tensorflow::external_mhlo_includes",
+        "MhloToStandard",
```

Comment:
This isn't a typo? Bazel one is SCF and this is Standard

Reply:
Not a typo :(